test-end-to-end/webapp-cypress/cypress/e2e/001_002_add_invalid_content.cy.js

```diff
@@ -0,0 +1,19 @@
+describe('001_002_add_invalid_content', () => {
+ it('Start Ustad Test Server ', () => {
+  // Start Test Server
+    cy.ustadStartTestServer()
+})
+  it('Admin user create a course and add members to the course', () => {
+
+  // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass',{timeout:8000})
+  cy.contains("Library").click()
+  cy.contains("button","Content").click()
+  cy.get('#new_content_from_file').click({force: true})
+  cy.get('input[type="file"]')
```

Comment:
Is force needed here because it is invisible? If so, let's put that as a comment.